package.json

```diff
@@ -1,24 +1,36 @@
 {
   "name": "daostack-arc",
-  "version": "0.0.0-alpha.19",
+  "version": "0.0.0-alpha.25",
   "description": "A platform for building DAOs",
+  "files": [
+    "contracts/",
+    "docs/",
+    "build/contracts/",
+    "migrations/",
+    "test/",
+    "truffle.js",
+    "tsconfig.json"
+  ],
   "scripts": {
-    "test": "truffle test",
-    "ganache": "ganache-cli -l 6500000 --account=\"0x0191ecbd1b150b8a3c27c27010ba51b45521689611e669109e034fd66ae69621,9999999999999999999999999999999999999999999\" --account=\"0x00f360233e89c65970a41d4a85990ec6669526b2230e867c352130151453180d,9999999999999999999999999999999999999999999\" --account=\"0x987a26abca7432016104ce2f24ce639340e25afe06ac69f68791399e7a5d1028,9999999999999999999999999999999999999999999\" --account=\"0x89af34b1b7347834048b99423dad174a14bf14540d720d72c16ae92e94b987cb,9999999999999999999999999999999999999999999\" --account=\"0xc867be647eb2bc51e4c0d61066859875cf3634fe949b6f5f85c69ab90e485b37,9999999999999999999999999999999999999999999\" --account=\"0xefabcc2377dee5e51b5a9e65a3854aec85fbbec3cb584d8ad4f9679869fb33c6,9999999999999999999999999999999999999999999\"",
+    "test": "./node_modules/.bin/run-with-testrpc --testrpc-cmd ganache-cli --gasLimit 6500000 'truffle compile && truffle migrate && truffle test'",
     "start": "pm2 start truffle -- serve",
     "lint": "eslint .",
     "lint --fix": "eslint --fix .",
     "solium": "solium --dir ./contracts/",
-    "truffle": "truffle"
+    "truffle": "truffle",
+    "build": "rimraf build && truffle compile"
   },
   "devDependencies": {
+    "@types/bignumber.js": "^4.0.3",
     "babel-cli": "^6.26.0",
     "babel-eslint": "^8.1.2",
     "babel-plugin-syntax-async-functions": "^6.13.0",
     "babel-polyfill": "^6.26.0",
     "babel-preset-es2015": "^6.24.1",
     "babel-register": "^6.26.0",
     "bignumber.js": "^5.0.0",
+    "bip39": "^2.4.0",
```

Comment:
we dont need this anymore, this is for arc-js

Reply:
Fixed at d4aa48d